<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Participants</class>
 <widget class="QWidget" name="Participants">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Participants</string>
  </property>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addParticipantBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add participant</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chooseParticipantBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose participant:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveChoicesBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save choices</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="genderGroupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender:</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="maleRB">
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="femaleRB">
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="participantsListWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>351</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
